src/test/java/Homework/DropdownCheckboxAlertTest.java

```diff
@@ -0,0 +1,76 @@
+package Homework;
+
+import base.BaseTest;
+import com.github.justthieenj.elements.CustomSelect;
+import com.github.justthieenj.utils.Utils;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.annotations.Test;
+
+public class DropdownCheckboxAlertTest extends BaseTest {
+
+    @Test
+    public void dropdownInteraction() {
+        dr.get("https://the-internet.herokuapp.com/");
+        dr.findElement(By.cssSelector("#content [href$='dropdown']")).click();
+        Select dropdown = new CustomSelect(dr.findElement(By.cssSelector("#dropdown")));
+        dropdown.selectByVisibleText("Option 2");
```

Comment:
use `Assert` to verify if the test is meet the expectation

Reply:
resolved